src/instructlab/eval/mt_bench_branch_generator.py

```diff
@@ -23,6 +23,9 @@ def get_file_paths(directory):
     logger.debug(locals())
     file_paths = []
     for root, _, files in os.walk(directory):
+        if "knowledge" in root:
+            logger.debug("skipping any knowledge directory")
```

Comment:
Maybe make this: Skipping knowledge directory {root}

Reply:
Worth making this a `warning` over `debug`?